src/client/src/orangehrmPerformancePlugin/pages/ReviewSearch.vue

```diff
@@ -350,7 +351,9 @@ export default {
           },
         };
       } else if (row.statusName === 'Inactive') {
+        //TODO:: Change to Id
```

Comment:
why TODO?

Reply:
We are to remove the use of status names and use ids instead. Change to be added in the next PR